commercial/app/controllers/commercial/ContentApiOffersController.scala

```diff
@@ -123,17 +123,22 @@ class ContentApiOffersController(contentApiClient: ContentApiClient, capiAgent:
 
   private def renderNative(format: Format, isMulti: Boolean) = Action.async { implicit request =>
 
-    retrieveContent().map(_.toList.headOption).map(
-      {
-        case None => Cached(componentNilMaxAge){ jsonFormat.nilResult }
-        case Some(content) => {
-          val capiSingle = CapiSingle.fromContent(content)
-          Cached(60.seconds) {
-            JsonComponent(capiSingle)
-          }
+    retrieveContent().map {content =>
```

Comment:
The list is up to 4 regardless is it not? It initially looks up eight `lookup.latestContentByKeyword(keyword, 8)`, and then trims that to four `(specific ++ latestByKeyword.filter(_.trail.trailPicture.nonEmpty)).distinct take 4`.

So do we not need `isMulti` to decide whether to take all or just the head?